client/src/App.js

```diff
@@ -33,6 +34,8 @@ const App = () => (
 			<Route path="/grant-access" element={<GrantAccessForm />} />
 			<Route path="/dashboard" element={<Dashboard />} />
 			<Route path="/dashboard/page/:pageNumber" element={<TendersList />} />
+			<Route path="/bidding" element={<BidList />} />
```

Comment:
What is this page?

Reply:
On “My Tender” page, Buyer can click on Tender ID or Tender Title to enter the details page. So it showing the bidding list submited for a tender.

Reply:
That's `/bidding/:tenderId`. What's `/bidding`?